README.md

```diff
@@ -82,10 +82,10 @@ credentials for configured AWS IAM roles.
 * Features
     * [Service Identity X.509 Certificates - Copper Argos](docs/copper_argos.md)
 * Developer Guide
-    * [Centralized Access Control](docs/dev_centralized_access.md)
+    * [Centralized Access Control](docs/decent_authz_flow.md)
```

Comment:
Thanks for the PR. This should be cent_authz_flow

Reply:
oops...
fixed: